src/Kconfig.coap

```diff
@@ -143,3 +143,8 @@ config GOLIOTH_COAP_MAX_PATH_LEN
     help
         Maximum length of a CoAP path (everything after
         "coaps://coap.golioth.io/").
+
+config LIBCOAP_ENABLE_IPV6_SUPPORT
```

Comment:
Since this is specific to ESP-IDF, let's keep in the port folder (`port/esp_idf/components/golioth_sdk/Kconfig`)